src/main/java/frc/robot/commands/WriteFiringSolutionAtCurrentPos.java

```diff
@@ -33,7 +33,9 @@ public void initialize() {
       // Calculates magnitude from x and y vals
       shotMag = rawTranslation.getDistance(FiringSolutionHelper.getSpeakerTranslation2d());
       FiringSolution solution =
-          new FiringSolution(shotMag, shotAngle, outtake.getTargetRPS(), outtake.getPivotTarget());
+          // Doesn't matter if you log top or bottom target RPS to JSON.
```

Comment:
this is fine for now but should be changed later (so that firing solutions can support top and bottom independently)